test/src/org/labkey/test/pages/cds/GroupDetailsPage.java

```diff
@@ -0,0 +1,53 @@
+package org.labkey.test.pages.cds;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.labkey.test.util.Ext4Helper;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class GroupDetailsPage extends LabKeyPage<LabKeyPage.ElementCache>
```

Comment:
Wrong generic here (probably why `elementCache()` wasn't working for you. These can be tricky so you should use the test file templates we have for IntelliJ (https://www.jetbrains.com/help/idea/settings-file-and-code-templates.html).
```suggestion
public class GroupDetailsPage extends LabKeyPage<GroupDetailsPage.ElementCache>
```